tests/Foundation/FoundationAliasLoaderTest.php

```diff
@@ -20,4 +20,10 @@ public function testGetInstanceCreatesOneInstance()
         $loader = AliasLoader::getInstance(['foo' => 'bar']);
         $this->assertSame($loader, AliasLoader::getInstance());
     }
+
+    // unregister dummy not existing classes
```

Comment:
it needs to be phpdoc then, we don't allow comments of that style outside methods